src/other/Table.js

```diff
@@ -621,7 +623,14 @@
             }
             return context._currentSortOrder * -1;
         });
-        this.render();
+        return this;
+    };
+
+    Table.prototype.headerClick = function (column, idx) {
```

Comment:
should change the order just to make it more clean since idx is used by default, but that might be just pulling hairs ... so you dont need to do headerClick(null, idx);